discord-scripts/invite-management.ts

```diff
@@ -22,10 +29,34 @@ async function createInvite(
   }
 }
 
+async function listInvites(discordClient: Client, robot: Robot): Promise<void> {
+  discordClient.guilds.cache.forEach(async (guild) => {
+    const fetchInvites = await guild.invites.fetch().catch((error) => {
+      robot.logger.error(
+        `Failed to fetch invites for guild ${guild.name}: ${error}`,
+      )
+    })
```

Comment:
When we use `async`/`await`, let's use `try`/`catch` rather than `.catch` behavior. It'll clean up the control flow in this whole function.